src/Raven.Server/Documents/TombstoneCleaner.cs

```diff
@@ -184,6 +229,8 @@ internal TombstonesState GetState(bool addInfoForDebug = false)
                         }
                     }
                 }
+
+                CreateWarningIfThereAreBlockingTombstones(tombstoneCollections);
```

Comment:
Let's wrap it, with `try... catch` and log in case of an exception.
This is a warning that shouldn't fail the entire tombstone cleaning process.

Reply:
done.